projects/1.19/assets/croparia/croparia/lang/zh_cn.json

```
@@ -0,0 +1,455 @@
+{
+  "item.croparia.rf_meter": "RF仪",
+  "item.croparia.flour": "面粉",
+  "item.croparia.obsidian_axe": "黑曜石斧头",

```

```suggestion
  "item.croparia.obsidian_axe": "黑曜石斧",
```
原版的固有名词应与Minecraft原版一致，如有特殊用法请指出。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **06.专有名词** 一节。
你也可以通过上方评论中，bot生成的对比（diff）文件或 [Minecraft 译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96) 以获取更多的原版翻译帮助。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。